app/build.gradle

```diff
@@ -282,9 +283,9 @@ dependencies {
     compileOnly Dependencies.daggerAssistedInject
     kapt Dependencies.daggerAssistedInjectProcessor
 
-    implementation(Dependencies.crashlytics) {
-        transitive = true
-    }
+//    implementation(Dependencies.crashlytics) {
```

Comment:
Please remove entirely instead of commenting out.

Reply:
Sure thing!